src/bindings.jl

```diff
@@ -140,15 +140,26 @@ function mark_binding!(x::EXPR, val=x)
     return x
 end
 
-
-function mark_parameters(sig::EXPR)
-    signame = CSTParser.rem_where_subtype(sig)
-    if CSTParser.iscurly(signame)
-        for i = 2:length(signame.args)
-            mark_binding!(signame.args[i])
+function mark_parameters(sig::EXPR, params = String[])
+    if CSTParser.issubtypedecl(sig)
+        mark_parameters(sig.args[1], params)
+    elseif iswhere(sig)
+        for i = 2:length(sig.args)
+            x = mark_binding!(sig.args[i])
+            push!(params, valof(bindingof(x).name))
```

Comment:
```suggestion
            val = valof(bindingof(x).name)
            if val isa String
                push!(params, val)
            end
```
`val` can be `nothing` in certain cases (couldn't repro it, but noticed it while testing).

Reply:
^@zacln